e2e/tests/migration_snapshot.feature

```diff
@@ -0,0 +1,51 @@
+# NOTE: The @oneshot & @manual tags allows this feature to be
+# excluded from any wildcard feature file execution (e.g. *.feature).
+#
+# The @manual tag indicates that a given feature depends on some non-automated
+# setup which MUST be performed manually, prior to running the feature.
+#
+# The @oneshot tag indicates that a given feature is non-idempotent with respect
+# to its impact on the network state. In such cases, a complete network reset
+# is required before running these features again.
+@oneshot @manual
+Feature: MorseAccountState Import
+
+  Scenario: Authority generates and imports MorseAccountState
+    # TODO_UPNEXT(@bryanchriswhite, #1034): Print a link to the latest liquify snapshot if no local state exists.
+    Given a local Morse node persisted state exists
+    # TODO_MAINNET: Replace 1000 with the published "canonical" export/migration/cutover height.
+    When the authority executes "pocket util export-genesis-for-reset 1000 poktroll" with stdout written to "morse_state_export.json"
+    Then a MorseStateExport is written to "morse_state_export.json"
+
+    When the authority executes "poktrolld tx migration collect-morse-accounts morse_state_export.json morse_account_state.json"
+    Then a MorseAccountState is written to "morse_account_state.json"
+
+    Given no MorseClaimableAccounts exist
+    And the MorseAccountState in "morse_account_state.json" is valid
+    When the authority executes "poktrolld tx migration import-morse-accounts --from=pnf --grpc-addr=localhost:9090 morse_account_state.json"
+    Then the MorseClaimableAccounts are persisted onchain
+
+  Scenario:
+    Given a Morse node snapshot is available
+    And the authority sucessfully imports MorseAccountState generated from the snapshot state
+    # TODO_INCOMPLETE: Ensure the liquify Morse snapshot includes known Morse
+    # private keys such that valid claim signatures can be generated for testing.
+    And Morse private keys are available in the following actor type distribution:
+      | non-actor | application | supplier |
+      | 2         | 2           | 2        |
+
+    When a Morse account-holder claims as a new non-actor account
+
```

Comment:
Even if its incomplete, can you add comments of what you had in mind?

Reply:
This is it, the reason it's incomplete is because the step definitions aren't all implemented and some of steps need to be updated/rephrased (e.g.: https://github.com/pokt-network/poktroll/pull/1133/files/87756bb7f2194c1e275cdb971138feaa334c365f#r2006790019).